src/world_interface/simulator_world.cpp

```diff
@@ -7,6 +7,11 @@
 World world;
 std::pair<double, double> cmdVel(0, 0);
 
+// gps coords of UW
+static double ORIGIN_LAT = 47.6553;
+static double ORIGIN_LON = -122.3035;
```

Comment:
Small nitpick, should probably make these `constexpr`
```suggestion
constexpr static double ORIGIN_LAT = 47.6553;
constexpr static double ORIGIN_LON = -122.3035;
```